bin/setup_dot1x

```diff
@@ -3,14 +3,14 @@
 ROOT=$(dirname $0)/..
 cd $ROOT
 
-sudo rm -rf inst/faucet/faucet_dot1x
-mkdir -p inst/faucet/faucet_dot1x
+#sudo rm -rf inst/faucet/daq-faucet-dot1x
+mkdir -p inst/faucet/daq-faucet-dot1x
 
-cp misc/faucet_dot1x.yaml inst/faucet/faucet_dot1x/faucet.yaml
+cp misc/faucet_dot1x.yaml.nodot inst/faucet/daq-faucet-dot1x/faucet.yaml
```

Comment:
It's still a yaml file so should end in .yaml -- faucet_nodot1x.yaml?

Reply:
When we copy it over, it is renamed to a .yaml extension. Should I still change it?

Reply:
yes -- b/c the "misc/faucet_dot1x.yaml.nodot" file itself is incorrect. E.g., if I look for "all .yaml files" it won't show up, and editors will likely be confused as to what it is.

Reply:
Yup. changed that.